docs/tutorials/portal/103/portal-103-4.rst

```diff
@@ -69,10 +70,13 @@ Spatial constraints are applied to the ``FROM`` table, not the ``JOIN`` table.
         AND cv.expMidptMJD > 60631 AND cv.expMidptMJD < 60637
         AND src.band = 'i'
 
+If a lightcurve appears in the left panel instead of a coverage HIPS map as shown in Figure 1, click the tab that says “Coverage” to display the HIPS map.
```

Comment:
This statement is currently floating after step 3, and we don't leave floating statements.

It only makes sense to have it as part of step 4, which reviews the results. In this case, it would make sense to put this information in the caption for Figure 1, which appears after step 4.